views/tag/edit_form.html

```
@@ -114,6 +114,27 @@
+        <!-- Audio Player -->
+        <div class="flex items-center gap-2">
+          <a href="/stream?path={{urlEncode .Track.Path}}" download="{{pathBase .Track.Path}}"
+             class="inline-flex items-center gap-1.5 px-2.5 py-1 rounded-full text-xs font-semibold bg-gray-100 dark:bg-gray-700 text-gray-600 dark:text-gray-300 border border-gray-300 dark:border-gray-600 hover:bg-gray-200 dark:hover:bg-gray-600 transition-colors">
+            <i class="fas fa-download fa-xs"></i> Download
+          </a>
+          <div class="audio-player flex items-center gap-2 flex-1">
+            <button type="button" class="play-btn flex-shrink-0 w-7 h-7 rounded-full bg-gray-200 dark:bg-gray-600 hover:bg-gray-300 dark:hover:bg-gray-500 flex items-center justify-center transition-colors"
+                    onclick="toggleAudioPlayer(this)">
+              <i class="fas fa-play fa-xs text-gray-700 dark:text-gray-200"></i>
+            </button>
+            <input type="range" min="0" max="100" value="0" step="0.1"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)